grafana_pandas_datasource.py

```diff
@@ -149,7 +149,7 @@ def _series_to_annotations(df, target):
 def _series_to_response(df, target):
     if df.empty:
         return {'target': '%s' % (target),
-                'datapoints': []}
+                'datapoints': zip(values, timestamps)}
```

Comment:
Hi @FatmaMagdy,

when looking at this more closely, I believe this is the code path where the dataframe is empty, so `values` and `timestamps` will be undefined here.

When looking at the whole function `_series_to_response()`:
https://github.com/panodata/grafana-pandas-datasource/blob/0686d7a07af50ac1b24fe1862fa49f3bb6c98dc3/grafana_pandas_datasource.py#L149-L164

we can see that `zip(values, timestamps)` will be returned as `datapoints` when there is actually data within the dataframe.

Did your update have any other special purpose which might currently not be obvious to me?

With kind regards,
Andreas.